commands/audit/auditparams.go

```diff
@@ -7,14 +7,17 @@ import (
 
 type AuditParams struct {
 	xrayGraphScanParams *services.XrayGraphScanParams
```

Comment:
This can be removed, this is not used anymore now that you added commonGraphScanParams